docs/investor.md

```diff
@@ -37,7 +37,7 @@ This function must return either 1) an `InvestorOpportunity` or 2) `undefined` t
 
 ### `findByUnderlyingAssetId: (assetId: string) => Promise<Array<InvestorOpportunity<TxType, MetaData>>>`
 
-`assetId` is a `AssetId` type as defined in `@shapeshiftoss/caip`. `AssetId` is a `string` formatted per the CAIP-19 specification.
+`assetId` is a `AssetId` type as defined in `@shapeshiftmonorepo/caip`. `AssetId` is a `string` formatted per the CAIP-19 specification.
```

Comment:
q: I know this was renamed because vite was not happy with installing `@shapeshiftoss` packages do we want to follow-up on a solution to keep leveraging our monorepo packages under their existing name (possibly by making this a *real* monorepo)?

Part of the reason I'm asking is because of namesquatting and risks involved with that

Reply:
Finally I think if we want to find a solution on this, best would be to find it now, or we will again change all the repository files

Reply:
I'm actually more than happy with your pragmatic solution of making it work first and making it better then, definitely a lot going on in this PR already to make it worth add more things to.
Think that's definitely not a now thing unless the solution is really straightforward and you're aware of it already, but would indeed be nice to capture it and start spiking on a solution to this, assuming it's actually possible!

Reply:
Finally I've been able to step back and use the same packages, maybe I didn't understand something when starting the PR, now that everything is gucci, it works as intended! So no more changes on this part! in [5111fe3](https://github.com/shapeshift/web/pull/8924/commits/5111fe34bfb62abf21804bf7cc91d3ea060193e5)